VideoLocker/src/main/java/org/edx/mobile/http/OfflineRequestInterceptor.java

```diff
@@ -27,16 +31,18 @@ public OfflineRequestInterceptor(Context context, int maxStaleTime){
     }
 
     @Override
-    public void intercept(RequestInterceptor.RequestFacade request) {
+    public Response intercept(Chain chain) throws IOException {
+        final Request.Builder requestBuilder = chain.request().newBuilder();
         if (isNetworkConnected()) {
             //TODO? should we specify caching here?
 //            int maxAge = 60; // read from cache for 1 minute
-//            request.addHeader("Cache-Control", "public, max-age=" + maxAge);
-            request.addHeader("Cache-Control", "public");
+//            builder.header(("Cache-Control", "public, max-age=" + maxAge);
```

Comment:
Variable name has been updated in commit c4b9f2bb455b9d612ca88aeb3b2d65576a356b5d.